README.md

```diff
@@ -1 +1,102 @@
+## Using Git in VS Code
+In this guide I will tell you how you, yes you could use git through VS code making your life significantly easier than before. Am I saying you will never have to use console technically as far as VS code is concerned yes with pushing and pulling. You do not have to use console except when you would need to roll back after pushing changes, let’s get started.
```

Comment:
You write as you speak. This is an anti-pattern in formal writing. Prefer shorter sentences and less commas.

"This guide will teach you how to use Git in VS Code to make your life easier. By the time you are done, you will not need to rely on using console commands to manage your Git workflow. Let's get started."